### PR TITLE
add private fields and accessors convention

### PR DESCRIPTION
**Pull request by steven-xu-lf**
issue -> #37

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <artifactId>lf-api-client-core</artifactId>
   <packaging>jar</packaging>
   <name>Laserfiche API Client Core</name>
-  <version>1.0.0-SNAPSHOT</version>
```
Comment (lf-jason-jia) [inline pom.xml]: Is this intentional?

File: pom.xml [hunk 2]
```diff
@@ -201,6 +201,16 @@
       <artifactId>nimbus-jose-jwt</artifactId>
       <version>9.22</version>
     </dependency>
+    <dependency>
```
Comment (lf-jason-jia) [inline pom.xml]: Do we need these new dependencies?

File: src/main/java/com/laserfiche/api/client/model/GetAccessTokenResponse.java [hunk 3]
```diff
@@ -1,17 +1,120 @@
 package com.laserfiche.api.client.model;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
```
Comment (lf-jason-jia) [inline src/main/java/com/laserfiche/api/client/model/GetAccessTokenResponse.java]: Now I see why have some new dependencies. I am sorry but the client core library isn't auto generated. Can we just hand-write this class (you can just remove the `@javax.annotation.Generated` annotation and stuff like `@Schema(description = "")` and the unnecessary `equals()`, `toString()`, `hashCode()`.

**Review by lf-jason-jia — CHANGES_REQUESTED** [review]
See comments, thanks!

File: pom.xml (hunk 1 above)
Comment (steven-xu-lf) [reply pom.xml]: yes, but i can change it back if you want

File: pom.xml (hunk 2 above)
Comment (steven-xu-lf) [reply pom.xml]: removed

**Review by lf-jason-jia — APPROVED** [review]

**Pull request merged**